.sphinx/_templates/footer.html

```diff
@@ -1,31 +1,33 @@
 {# ru-fu: copied from Furo, with modifications as stated below #}
 
 <div class="related-pages">
-  {% if next -%}
-    <a class="next-page" href="{{ next.link }}">
-      <div class="page-info">
-        <div class="context">
-          <span>{{ _("Next") }}</span>
+  {% if sequential_nav != "none" -%}
```

Comment:
Can you add some comments to make it clear these parts are changed? See the comment at the top.

Reply:
Sure. To keep it consistent, I marked all comments but the first as "mod". This involved changing 2 of your existing ones.